[DOC] Add landing page for simplified exploration (#93515)
grafana/grafana: docs/sources/explore/simplified-exploration/_index.md

title: Simplified exploration
menuTitle: Simplified exploration
weight: 100
hero:
  title: Simplified exploration with the Explore apps
  level: 1
  width: 100
  height: 100
  description: Use Explore Profiles to investigate and identify issues using profiling data.
cards:
  title_class: pt-0 lh-1
  items:
    - title: Explore Metrics
      href: ./metrics/

>       description: Quickly find related metrics with just a few simple clicks, without needing to write PromQL queries to retrieve metrics.

Author: ```suggestion
      description: Quickly find related metrics with a few clicks, without needing to write PromQL queries to retrieve metrics.
```

      height: 24
    - title: Explore Logs
      href: ./logs/

>       description: Visualize log volumes to easily detect anomalies or significant changes across labels and fields over time.

Author: ```suggestion
      description: Visualize log volumes to easily detect anomalies or significant changes over time, without needing to compose LogQL queries.
```

      height: 24
    - title: Explore Traces
      href: ./traces/

>       description: Use Rate, Errors, and Duration (RED) metrics derived from traces to investigate to understand errors and latency issues within complex distributed systems.

Author: ```suggestion
      description: Use Rate, Errors, and Duration (RED) metrics derived from traces to investigate and understand errors and latency issues within complex distributed systems.
```
